src/flows/shared/steps/OrganizeCollection/Sidebar/EmptySidebar.tsx

```diff
@@ -0,0 +1,15 @@
+import { BaseM, TitleDiatypeL } from 'components/core/Text/Text';
+import { VStack } from 'components/core/Spacer/Stack';
+
+type Props = {
+  chain: string;
+};
+
+export function EmptySidebar({ chain }: Props) {
+  return (
+    <VStack grow align="center" justify="center">
```

Comment:
maybe we can show a slightly different message if there aren't any matching search results here? like "No NFTs matching your search query"

![Screen Shot 2022-09-12 at 20 27 42](https://user-images.githubusercontent.com/80802871/189642276-7b9e124b-07ca-40b6-9e87-13d4445d61f0.png)